scripts/perftests/PerformanceTestUtilities.ps1

```diff
@@ -0,0 +1,214 @@
+# Contains all the utility methods used by the performance tests.
+
+    # Downloads a nuget exe in the given directory.
+    # The exe will be found at $directory/$version/nuget.exe
+    function DownloadNuGetExe([string]$version, [string]$downloadDirectory)
```

Comment:
This function isn't [used](https://github.com/NuGet/NuGet.Client/pull/2357/files#diff-c1abeb59a21c178f7360dc2ac1d5ac49R1) anywhere.  Remove it?